src/state/apis/limit-orders/limitOrderApi.ts

```diff
@@ -86,7 +86,15 @@ export const limitOrderApi = createApi({
                   `${baseUrl}/${network}/api/v1/account/${account}/orders?limit=1000`,
                 )
 
-                return result.data.map(order => {
+                // CowSwap limit and spot orders API is the same, so we need to filter out spot orders
+                // there are no parameters to filter from their API, they are filtering after fetching
+                // on their interface as it's some custom metadata they add
+                const limitOrders = result.data.filter(order => {
+                  const appData = order.fullAppData ? JSON.parse(order.fullAppData) : null
+                  return !appData?.metadata.orderClass.orderClass.includes(OrderClass.MARKET)
+                })
```

Comment:
preferably-blocking: don't JSON.parse() as this gets really expensive on large sets.

What about 
```suggestion
                const limitOrders = result.data.filter(order => order.class === OrderClass.LIMIT)
```

FYI this is how limit and market orders will be returned as respectively:

<img width="152" alt="image" src="https://github.com/user-attachments/assets/4e9b217f-45da-4290-b5ba-f04ef31b8a0b" />
<img width="141" alt="image" src="https://github.com/user-attachments/assets/78a48e4a-db71-4667-adce-40a4fae03d94" />

Reply:
Disregard me, we unfortunately need this

Reply:
Typed in [3592597](https://github.com/shapeshift/web/pull/9199/commits/3592597661176a6ca175a9465d06fe4fd2d62b16)